package.json

```diff
@@ -2,10 +2,11 @@
   "name": "hyperx",
   "version": "2.3.2",
   "description": "tagged template string virtual dom builder",
-  "main": "index.js",
+  "main": "dist/hyperx.js",
   "scripts": {
     "test": "tape test/*.js",
-    "coverage": "covert test/*.js"
+    "coverage": "covert test/*.js",
+    "prepack": "webpack"
```

Comment:
using browserify with tinyify seems simpler than a webpack config file in the root just for umd:

```json
"scripts": {
  "prepublish": "browserify -s hyperx -p tinyify index.js -o dist/hyperx.js"
}
```

and a bit more in line with other choo-adjacent projects :)